### PR TITLE
Update roadmap for completed work

### PR DESCRIPTION
**Pull request by samuelhwilliams**

### CODE REVIEW

**Review by whpearson — APPROVED** [review]
Changes make sense with what I know about the features released.

File: app/templates/views/guidance/features/roadmap.html [hunk 1]
```diff
@@ -15,39 +15,34 @@ <h1 class="heading-large">Roadmap</h1>
...
   <p class="govuk-body">The roadmap is only a guide. It does not cover everything we do, and some things may change.</p>
   <p class="govuk-body">You can <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.support')}}">contact us</a> if you have any questions about the roadmap, want to get involved in user research or have suggestions for new features.</p>
 
   <h2 class="heading-medium" id="things-we-are-working-on">Things we’re working on now</h2>
 
   <ul class="govuk-list govuk-list--bullet">
-    
-    <li>Launch a regular email about new features and improvements to GOV.UK Notify</li>
-    <li>Let services add QR codes to letter templates</li>   
+
```
Comment (karlchillmaid) [inline app/templates/views/guidance/features/roadmap.html:30]:
```suggestion
```

File: app/templates/views/guidance/features/roadmap.html [hunk 2]
```diff
@@ -29,23 +28,21 @@ <h2 class="heading-medium" id="things-we-are-working-on">Things we’re working
...
     <li>Continue to improve our front-end code to meet our <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.accessibility_statement')}}">targets for accessibility</a></li>
-    <li>Support services to write better notifications</li>
     <li>Migrate our database from <a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/">GOV.UK PaaS</a> to one hosted using <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/rds/">Amazon RDS</a></li>
     <li>Provide more detailed reporting and usage data for services</li>
   </ul>
 
   <h2 class="heading-medium" id="things-we-will-do-later">Things we’ll do later</h2>
 
-  <h3 class="heading-small" id="april-to-june">August to October 2023</h3>
+  <h3 class="heading-small" id="april-to-june">October to December 2023</h3>
```
Comment (samuelhwilliams) [inline app/templates/views/guidance/features/roadmap.html:40]:
```suggestion
  <h3 class="heading-small" id="october-to-december">October to December 2023</h3>
```

🤔

Comment (karlchillmaid) [reply app/templates/views/guidance/features/roadmap.html:40]: I think the IDs exist because we used to have several quarters on the page.

**Comment by samuelhwilliams** [comment]
superceded

**Pull request closed without merging**